<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportWidget</class>
 <widget class="QWidget" name="ImportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="m_tbtn_browse">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>81</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Excel文件：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_edt_path">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>20</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_radio_mac">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>写MAC</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_radio_rx">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RX</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_radio_rf">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RF</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>64</y>
     <width>81</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择数据库：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmb_table">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>120</y>
     <width>51</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择表：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_import">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_back">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
